ippserver/__main__.py

```diff
@@ -75,5 +75,8 @@ def main(args):
 		behaviour_from_args(args))
 	run_server(server)
 
-if __name__ == "__main__":
+def _init():
```

Comment:
What's the advantage of this? No global variables get leaked, and we already have a `main`

Reply:
it is installable with setuptools. You get a script which executes your code.

Reply:
That's a pretty neat feature, I like that.

I can't really justify why I didn't put the argparse in the main function. It's probably worth merging init and main, so there's just one obvious entry-point to the project.